moonstone/normalization/counts/random_selection.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, df: pd.DataFrame, threshold: int = None, random_seed: int = 2
     def _randomly_select_counts(self, column_name: str):
         np.random.seed(self.random_seed)  # set the random seed
         counts = self.df[column_name]
-        if counts.sum() <= self.threshold:
+        if counts.sum() <= self.threshold + 1:
```

Comment:
why not < self.threshold ?

Reply:
This is because of the confusion when handling counts that are actually floats. Since the threshold is force to be an `int` this can leads to errors